aws-s3/scripts-ccao-data-raw-us-east-1/spatial-point_source_air_noise.R

```diff
@@ -0,0 +1,168 @@
+# Create model of point-source inverse-square airport noise falloff
+
+# Matt Jackson, October 2023
+
+library(arrow)
+library(aws.s3)
+library(ccao)
+library(DBI)
+library(geoarrow)
+library(ggplot2)
+library(ggpubr)
+library(glue)
+library(gstat)
+library(here)
+library(noctua)
+library(stars)
+library(sf)
+library(sp)
+# "The legacy packages maptools, rgdal, and rgeos, underpinning the sp package,
+# which was just loaded, will retire in October 2023.
+# Please refer to R-spatial evolution reports for details, especially
+# https://r-spatial.org/r/2023/05/15/evolution4.html.
+# It may be desirable to make the sf package available;
+# package maintainers should consider adding sf to Suggests:.
+# The sp package is now running under evolution status 2
+# (status 2 uses the sf package in place of rgdal)"
+library(tidycensus)
+library(tidyverse)
+library(tmap)
+# "Breaking News: tmap 3.x is retiring. Please test v4, e.g. with
+# remotes::install_github('r-tmap/tmap')"
+library(units)
+library(viridis)
+
+## READ DATA ===================================================================
+
+# Optionally enable query caching
+noctua_options(cache_size = 10)
+
+# Establish connection
+AWS_ATHENA_CONN_NOCTUA <- dbConnect(noctua::athena())
+print("noctua connection established")
+
+midway <- read_parquet(
+  "s3://ccao-data-warehouse-us-east-1/spatial/environment/midway_noise_monitor/midway_noise_monitor.parquet"
```

Comment:
**nitpick:** Both Midway and O'Hare have monitor tables in Athena, let's just use the Athena method of grabbing them for both datasets.